src/main/java/frc/robot/commands/auto/routines/TestAutoCommandGroup.java

```diff
@@ -1,16 +0,0 @@
-package frc.robot.commands.auto.routines;
-
-
-import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
-import frc.robot.commands.auto.MoveCommand;
-import frc.robot.commands.auto.TurnCommand;
-import frc.robot.subsystems.Drivetrain;
```

Comment:
this should not be deleted- we still need it

Reply:
how do i undelete it then

Reply:
duly undeleted